src/main/java/com/benmorant/game/holycraap/model/entity/Item.java

```diff
@@ -25,4 +26,90 @@
   private int slot;
 
   private double price;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JsonBackReference
+  private People people;
+
+  public Item() {}
+
+  /** Constructeur surchargé pour Item (moins l'ID, auto-généré. */
+  public Item(String name, int slot, double price, People people) {
+    this.name = name;
+    this.slot = slot;
+    this.price = price;
+    this.people = people;
+  }
+
+  public Integer getId() {
+    return id;
+  }
+
+  public void setId(Integer id) {
+    this.id = id;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public int getSlot() {
+    return slot;
+  }
+
+  public void setSlot(int slot) {
+    this.slot = slot;
+  }
+
+  public double getPrice() {
+    return price;
+  }
+
+  public void setPrice(double price) {
+    this.price = price;
+  }
+
+  public People getPeople() {
+    return people;
+  }
+
+  public void setPeople(People people) {
+    this.people = people;
+  }
+
+  /** Méthode equals pour Item. */
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof Item)) {
+      return false;
+    }
+    return id != null && id.equals(((Item) o).getId());
+  }
+
+  @Override
+  public int hashCode() {
+    return 31;
+  }
+
+  @Override
+  public String toString() {
+    return "Item{"
+        + "id="
+        + id
+        + ", name='"
+        + name
+        + '\''
+        + ", slot="
+        + slot
+        + ", price="
+        + price
+        + '}';
```

Comment:
Concatenating words like this is a bad practice and is (if not optimized by the compiler) bad in terms of performance.

Each concatenation will result in:
- a new string being allocated (during the operation) to store the result
- both strings to be traversed fully

Also it's not always the most readable, like here since the autoformatter decided it would print one operand per line.

My guideline is to generally keep string concatenation for readability of longer strings.

Here, you should have a look at `String#format` or `StringBuilder`. I find the former more readable, and I would recommend `StringBuilder` if you need to do a loop (which isn't the case here).